.chronus/changes/fix-patch-2024-5-13-3-36-28.md

```diff
@@ -0,0 +1,8 @@
+---
+# Change versionKind to one of: internal, fix, dependencies, feature, deprecation, breaking
+changeKind: fix
+packages:
+  - "@azure-tools/typespec-autorest"
+---
+
+Fix #383: Generate patch models for common-types
```

Comment:
```suggestion
Fix generate patch models for common-types
```
not sure this will render nicely in the changelog and chronus will auto link PRs already